spec/models/content_asset_spec.rb

```diff
@@ -25,5 +26,9 @@
       subject.validate
       expect(subject.errors[:avatar]).to include("can't be blank")
     end
+    it "requires an alt text for the asset" do
+      subject.validate
```

Comment:
Could extract `subject.validate` into a `before` block to reduce the repetition on this block of specs

Reply:
extraction complete